src/utils/imageCompress.ts

```diff
@@ -5,4 +5,6 @@ export enum AvailableForCompressExtension {
 }
 
 export const isCompressible = (fileName: string) =>
-    Object.keys(AvailableForCompressExtension).some((ext) => fileName.endsWith(`.${ext}`));
+    Object.values(AvailableForCompressExtension).some((ext) =>
```

Comment:
why we changed here keys to values?

Reply:
While doing test I found bug, and fixed it

Reply:
I think that it shouldn't be in the PR with test then